kover-jvm-agent/build.gradle.kts

```diff
@@ -23,17 +23,17 @@ plugins {
 }
 
 koverPublication {
-    description.set("Kover JVM instrumentation agent")
+    description = "Kover JVM instrumentation agent"
 }
 
 koverDocs {
-    docsDirectory.set("jvm-agent")
-    description.set("Kover JVM instrumentation agent")
+    docsDirectory = "jvm-agent"
+    description = "Kover JVM instrumentation agent"
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_6
-    targetCompatibility = JavaVersion.VERSION_1_6
+    sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
We are specifically compiling for the `1.6` version for wider support of different JVMs.
Therefore, it is better to leave `1.6`

Reply:
But unable to run `clean build` locally, it says `error: Source option 6, 7 is no longer supported. Use 8 or later. `
I reverted back to as it is before, let see CI Build results.